x-pack/plugins/transform/public/app/sections/transform_management/components/transform_list/expanded_row_messages_pane.tsx

```diff
@@ -110,11 +133,20 @@ export const ExpandedRowMessagesPane: React.FC<Props> = ({ transformId }) => {
         }
       ),
       width: '50%',
+      sortable: true,
     },
   ];
 
   const getPageOfMessages = ({ index, size }: { index: number; size: number }) => {
-    const list = messages;
+    let list = messages;
+    if (msgCount <= 500) {
```

Comment:
The limit `500` is used a couple of times in this file. Is it worth creating a `const` for it?

Reply:
Updated here https://github.com/elastic/kibana/pull/135047/commits/24f257a6a6ca53959885a9311171f6cb1b0b12dc